daemon/src/taker_cfd.rs

```diff
@@ -62,16 +59,14 @@ impl Actor {
         db: sqlx::SqlitePool,
         wallet: Wallet,
         oracle_pk: schnorrsig::PublicKey,
-        cfd_feed: CfdFeed,
+        cfd_feed_actor_inbox: watch::Sender<Vec<Cfd>>,
         order_feed_actor_inbox: watch::Sender<Option<Order>>,
         send_to_maker: Address<send_to_socket::Actor<wire::TakerToMaker>>,
         monitor_actor: Address<monitor::Actor<Actor>>,
         cfds: Vec<Cfd>,
     ) -> Result<Self> {
         let mut conn = db.acquire().await?;
-
-        // populate the CFD feed with existing CFDs
-        cfd_feed.update(&mut conn).await?;
+        cfd_feed_actor_inbox.send(load_all_cfds(&mut conn).await?)?;
```

Comment:
Given, that after the restart changes we now pass in the initial list of cfds you don't have to lead them from the db here, but could just clone them into the feed. (I don't mind much though, I think loading is not expensive)